### DISCUSSION TITLE
Call OpenAPI plugins having authentication

### DISCUSSION THREAD

**Discussion by amaltv** [microsoft/semantic-kernel#9783] [Q&A] [answered]
Do we have any examples on how to call OpenAPI plugins having authentication

**Comment by sophialagerkranspandey** [comment, marked as answer]
https://devblogs.microsoft.com/semantic-kernel/announcing-the-ga-release-of-the-openapi-plugin-for-net-and-python-in-semantic-kernel/ this article has a good overview. I'm also tagging @SergeyMenshykh, @dmytrostruk and @moonbox3 to further elaborate as well.

**Comment by moonbox3** [comment]
Hi @amaltv, are you using SK C#, Python, or Java?

**Comment by amaltv** [comment]
We are using python..

**Reply by moonbox3** [reply]
Thanks for the response. We don’t have a sample right now that shows how auth is used with an OpenAPI plugin in Python. I can put one together next week, and will let you know it’s available. 

For my understanding, what type of auth do you plan to use?
